Frontend/src/interfaces/IHeaderProps.ts

```diff
@@ -5,4 +5,6 @@ export interface IHeaderProps {
   setTasks: (allTasks: ITask[]) => void;
   selectedTasks: number[];
   taskHandler: () => void;
+  togglePending: boolean;
+  toggleCompleted: boolean;
```

Comment:
toggle is a verb, so I would expect `toggleCompleted` to update update some piece of state. 

I would change these names to `showCompleted` & `showPending`.

Reply:
Done.